starknet_devnet/blueprints/base.py

```diff
@@ -88,3 +88,13 @@ async def get_balance():
         "amount": balance,
         "unit": "wei"
     })
+
+@base.route("/get_predeployed_accounts", methods=["GET"])
```

Comment:
What do you think about keeping the verb out of the URL? Like, just having `"predeployed_accounts"`, similar to how getting account balance is done with GET /account_balance.

Reply:
Agree with that, what bothers me here is the underscore instead of the dash in the API endpoint